config/code_schools.yml

```diff
@@ -26,6 +26,17 @@
   va_accepted: false
   full_time: true
   hardware_included: false
+  
+  - name: Fullstack Academy
```

Comment:
@karla001 - I'm almost positive it's here, check the spacing. See the following stack trace from TravisCI;

`Error:
CodeSchoolsTest#test_it_exists:
Psych::SyntaxError: (/home/travis/build/OperationCode/operationcode/config/code_schools.yml): did not find expected key while parsing a block mapping at line 19 column 3
    lib/code_schools.rb:7:in `initialize'
    test/lib/code_schools_test.rb:5:in `new'
    test/lib/code_schools_test.rb:5:in `block in <class:CodeSchoolsTest>'`